app/src/main/kotlin/at/bitfire/davdroid/ui/webdav/WebdavMountsActivity.kt

```diff
@@ -122,81 +117,238 @@ class WebdavMountsActivity: AppCompatActivity() {
-            oldItem.mount.name == newItem.mount.name && oldItem.mount.url == newItem.mount.url &&
-            oldItem.rootDocument?.quotaUsed == newItem.rootDocument?.quotaUsed &&
-            oldItem.rootDocument?.quotaAvailable == newItem.rootDocument?.quotaAvailable
-    }) {
-        class ViewHolder(val binding: WebdavMountsItemBinding): RecyclerView.ViewHolder(binding.root)
-
-        val authority = context.getString(R.string.webdav_authority)
-
-        override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
-            val inflater = LayoutInflater.from(parent.context)
-            val binding = WebdavMountsItemBinding.inflate(inflater, parent, false)
-
-            return ViewHolder(binding)
+    @Composable
+    fun WebdavMountsContent(mountInfos: List<MountInfo>) {
+        val uriHandler = SafeAndroidUriHandler(this)
+
+        Scaffold(
+            topBar = {
+                TopAppBar(
+                    navigationIcon = {
+                        IconButton(
+                            onClick = ::finish
+                        ) {
+                            Icon(
+                                imageVector = Icons.AutoMirrored.Filled.KeyboardArrowLeft,
+                                contentDescription = null
+                            )
+                        }
+                    },
+                    title = { Text(stringResource(R.string.webdav_mounts_title)) },
+                    actions = {
+                        IconButton(
+                            onClick = { uriHandler.openUri(helpUrl().toString()) }
+                        ) {
+                            Icon(
+                                imageVector = Icons.AutoMirrored.Filled.Help,
+                                contentDescription = stringResource(R.string.help)
+                            )
+                        }
+                    }
+                )
+            },
+            floatingActionButton = {
+                FloatingActionButton(
+                    onClick = { startActivity(Intent(this, AddWebdavMountActivity::class.java)) }
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.Add,
+                        contentDescription = stringResource(R.string.webdav_add_mount_add)
+                    )
+                }
+            }
+        ) { paddingValues ->
+            LazyColumn(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(16.dp)
+                    .padding(paddingValues)
+                    .verticalScroll(rememberScrollState())
+            ) {
+                items(mountInfos, key = { it.mount.id }, contentType = { "mount" }) {
+                    WebdavMountsItem(it)
+                }
+                if (mountInfos.isEmpty()) {
+                    item(key = "empty", contentType = "text") {
+                        Text(
```

Comment:
This Text (and the text below) should be centered vertically like in the original layout.